src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonShadowAssassinNotification.kt

```diff
@@ -15,7 +15,7 @@ class DungeonShadowAssassinNotification {
     @SubscribeEvent
     fun onWorldBoarderChange(event: PacketEvent.ReceiveEvent) {
         if (!isEnabled()) return
-        if (DungeonAPI.dungeonFloor?.contains("3") == true && DungeonAPI.inBossRoom) return
+        if (DungeonAPI.dungeonFloor?.name?.contains("3") == true && DungeonAPI.inBossRoom) return
```

Comment:
This feels a bit cleaner and easier to understand at a glance.
```suggestion
        if (DungeonAPI.dungeonFloor in setOf(DungeonFloor.F3, DungeonFloor.M3) && DungeonAPI.inBossRoom) return
```